src/main/cli/cli.c

```diff
@@ -2703,6 +2707,58 @@ static void cliVtx(const char *cmdName, char *cmdline)
 
 #endif // VTX_CONTROL
 
+#ifdef USE_VTX_SMARTAUDIO
+static void cliVtxOpModel(const char *cmdName, char *cmdline)
+{
+    if (isEmpty(cmdline)) {
+        // If you change the model this value will be stale until reboot.
+        cliPrintLine(saDevice.willBootIntoPitMode ? "race" : "free");
+    } else {
+        bool freeModel = strncasecmp(cmdline, "free", 4) == 0;
+        bool pir = strncasecmp(cmdline, "pir", 3) == 0 || strncasecmp(cmdline, "race", 4) == 0;
+        bool por = strncasecmp(cmdline, "por", 3) == 0;
+        char *unsupported = "%s is unsupported by SA %s";
+
+        if (saDevice.version == 1) {
+            // V1 devices don't support pit mode.
+            if (!freeModel) {
+                cliPrintErrorLinef(cmdName, unsupported, cmdline, "1.0");
+            }
+            return;
+        }
+
+        int mode;
+
+        if (freeModel) {
+            // VTX should power up transmitting - unset PIR and POR bits.
+            mode = 0;
+        } else if (pir) {
+            mode = SA_MODE_SET_IN_RANGE_PITMODE;
+
+            if (saDevice.version >= 3 && !(saDevice.mode & SA_MODE_GET_PITMODE)) {
+                mode |= SA_MODE_CLR_PITMODE;
+            }
+        } else if (por) {
+            // V2.1 devices only supports PIR mode.
+            if (saDevice.version >= 3) {
+                cliPrintErrorLinef(cmdName, unsupported, cmdline, "2.1");
+                return;
+            }
+
+            mode = SA_MODE_SET_OUT_RANGE_PITMODE;
+        } else {
+            cliPrintErrorLinef(cmdName, "Invalid option");
+            return;
+        }
+
+        saSetMode(mode);
+
+        // Send queued mode to device.
+        vtxCommonProcess(vtxCommonDevice(), 0);
```

Comment:
The VTX task is suspended when using the CLI so I have to call `vtxCommonProcess` to send the `mode` value queued by `saSetMode` (otherwise it will be lost on exiting the CLI).